### PR TITLE
Stabilize Python codegen.

### PR DESCRIPTION
**Pull request by pgavlin**
Make sure all property casing conversions are computed prior to codegen.

### CODE REVIEW

**Comment by pgavlin** [comment]
Fixes https://github.com/pulumi/pulumi-terraform-bridge/issues/129.

**Comment by pgavlin** [comment]
Note that this is not quite a complete fix for the linked issue: it does not fix the instability in the version that is embedded in the schema. It does fix the Python codegen, though.

File: pkg/codegen/python/gen.go [hunk 1]
```diff
@@ -1199,14 +1185,32 @@ func GeneratePackage(tool string, pkg *schema.Package, extraFiles map[string][]b
 		_ = getMod(":config/config:")
 	}
 
-	scanResource := func(r *schema.Resource) {
+	scanResource := func(r *schema.Resource) error {
 		mod := getMod(r.Token)
 		mod.resources = append(mod.resources, r)
+
+		// Calculate casing tables. We do this up front because our docstring generator (which is run during
+		// genResource) requires them.
```
Comment (lukehoban) [inline pkg/codegen/python/gen.go]: This comment doesn't appear to be correct anymore - and does the comment point to any potential issue with this change?

Comment (pgavlin) [reply pkg/codegen/python/gen.go]: > This comment doesn't appear to be correct anymore

The comment is still correct. Which part seems wrong?

>  - and does the comment point to any potential issue with this change?

It pointed to the issue that these changes are fixing.

**Pull request merged**